packages/create-schemas/src/plugins/react-query-plugin/openapi-client.ts

```diff
@@ -0,0 +1,109 @@
+import { buildURL, serializeBody, mergeHeaders, type SerializeQueryOptions } from "./openapi-client-helpers.ts";
+
+interface OpenAPIClientOptions {
+    baseURL?: string;
+    query?: SerializeQueryOptions;
+    headers?: Record<string, string>;
+}
+
+export const internal_fetch = Symbol();
```

Comment:
A `Symbol` is used to create a method that isn't accessible by the consumer. This way, people can't use the client directly, as disscussed in our last meeting.